linkedListCycleII.py

```diff
@@ -0,0 +1,24 @@
+# Definition for singly-linked list.
+# class ListNode(object):
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution(object):
+    def detectCycle(self, head):
+        """
+        :type head: ListNode
+        :rtype: ListNode
+        """
+        slow, fast = head, head
+        while fast and fast.next:
+            fast = fast.next.next
+            slow = slow.next
+            if slow == fast:
+                break
+        if not fast or not fast.next:
```

Comment:
This is OK, but you can also define a flag as `false` before `while` loop.
`if slow == fast:`, mark `flag` as `true`.

Reply:
You commit messed up.
There is another pull request containing the same 3 files.